.appveyor.yml

```
@@ -47,7 +47,7 @@ install:
   - cinst gsutil
   - cinst openssl.light --version 1.1.1
   - python -m pip install -U pip setuptools wheel
-  - pip install -r requirements.txt
+  - pip install -e .
```

The tests are failing because we need to install all cloud deps:
```suggestion
  - pip install -e '.[all]'
```